### ISSUE TITLE
Use consistent types

### ISSUE THREAD

**Issue by cfriedt** [cisco/openh264#3141]
Throughout the code, int and int32_t are used interchangeably.

**Comment by huili2** [comment]
Agree. We have paid some effort on this, and think critical parameters (sensitive to 32/64 bit) have been specifically marked, while others are not so important.
If you find sensitive parameters not subjective to that, please inform us, or directly make contributions to that.
Thanks.
